Assets/Tests/InputSystem/TestActionsAsset.cs

```diff
@@ -0,0 +1,5 @@
+using UnityEngine;
+
+internal class TestActionsAsset : ScriptableObject
```

Comment:
Could this be a TestProjectWideActionsAsset instead?

Reply:
It's used by ProjectWideActions' test currently, but it's just a dummy `InputActionsAsset`, so it could be used elsewhere.

Reply:
All good then 👍